modules/position/src/main/native/android/dalvik/DalvikPositionService.java

```diff
@@ -85,36 +82,44 @@ public DalvikPositionService(Activity activity) {
         }
     }    
 
-    public void start() {
-        start(DEFAULT_PARAMETERS);
+    public void enableDebug() {
+        debug = true;
     }
-
-    public void start(Parameters parameters) {
-        Log.v(TAG, "DalvikPositionService, start called");
+    
+    public void start(long timeInterval, float distanceFilter, boolean backgroundModeEnabled) {
+        if (debug) {
+            Log.v(TAG, "DalvikPositionService, start called");
+        }
         if (running) {
             stop();
         }
         
-        this.parameters = parameters;
+        this.timeInterval = timeInterval;
+        this.distanceFilter = distanceFilter;
+        this.backgroundModeEnabled = backgroundModeEnabled;
         
         initialize();
         
         running = true;
     }
 
     public void stop() {
-        Log.v(TAG, "DalvikPositionService, stop called, quit looper");
+        if (debug) {
+            Log.v(TAG, "DalvikPositionService, stop called, quit looper");
```

Comment:
Since it's almost impossible for a developer to toggle debug on/off, we should assume debug is always false. Hence, I think it would be best to have those minimal debug messages about whether a positionservice is started/stopped at all should still be printed.

Reply:
Done